### ISSUE TITLE
Does this adapter work?

### ISSUE THREAD

**Issue by GermanBluefox** [pyatovsky/ioBroker.vera#1]
Do you think about repository?

**Comment by pyatovsky** [comment]
Yes, it works. But this is not a last version. What do you mean "Do you think about repository?"?

**Comment by GermanBluefox** [comment]
I mean, do you want to publish it for other users in official ioBroker
repository?

2018-04-10 20:50 GMT+02:00 pyatovsky <notifications@github.com>:

**Comment by pyatovsky** [comment]
Sure.
But it is not finished. Yes, it works, but has not been completed to the "logical finish".
I must finish it up at least to any "logical stage".
And a documentation must also be prepared.

**Comment by GermanBluefox** [comment]
Ok. Thank you.

2018-04-11 19:22 GMT+02:00 pyatovsky <notifications@github.com>:
